### ISSUE TITLE
Proposal: remove the Win32 frontend in favour of Qt

### ISSUE THREAD

**Issue by ajtribick** [CelestiaProject/Celestia#1199]
I don't think it makes sense given the limited amount of active developers and time to have a separate Win32 UI when we already have a full-featured cross-platform UI using Qt. I therefore propose we remove the Win32 frontend.

If there are (useful) features in the Win32 frontend that are not present in Qt, we should migrate them.

Thoughts? Objections?

**Comment by levinli303** [comment]
yeah I think win32 frontend probably has much obscure code that we don't understand and is very hard to maintain. also very hard to modernize it as it is written in raw win32.

the only good thing about it is that probably win32 will never go away as MS continues to support it

**Comment by 375gnu** [comment]
I raised my concerns about Qt a year and a half ago. Unfortunately they are still valid.

**Comment by SevenSpheres** [comment]
I'd be fine with dropping the Win32 frontend, since the Qt frontend is cross-platform and has more features. But see also #751.

**Comment by ajtribick** [comment]
I've raised #1202 to enable Qt6 support, including a workaround for the fullscreen menu issue on Windows. As far as I can tell, it seems to work (at least on Kubuntu 21.10 and Windows 10), but haven't got a Mac to test it on to see if there are issues there.
